### PR TITLE
Finds command error on no specified option

### PR DESCRIPTION
**Pull request by Denniszedead**

### CODE REVIEW

File: src/main/java/manageezpz/logic/commands/FindEmployeeCommand.java [hunk 1]
```diff
@@ -35,6 +35,7 @@
             + COMMAND_WORD + " " + PREFIX_EMAIL.getPrefix() + "alexyeo@google.com\n"
             + COMMAND_WORD + " " + PREFIX_NAME.getPrefix() + "Benson Chua " + PREFIX_PHONE.getPrefix() + "6123456 "
             + PREFIX_EMAIL.getPrefix() + "bensonc@gmail.com";
+    public static final String NO_OPTIONS = COMMAND_WORD + " needs at least 1 valid options\n";
```
Comment (alfredkohhh) [inline src/main/java/manageezpz/logic/commands/FindEmployeeCommand.java:38]: Thanks for adding this in per our discussion

File: src/main/java/manageezpz/logic/parser/FindEmployeeCommandParser.java [hunk 2]
```diff
@@ -106,4 +114,12 @@ private void addErrorMessage(String errorMessage) {
         hasError = true;
         this.errorMessage = this.errorMessage + errorMessage;
     }
+
+    /**
+     * Returns true if none of the prefixes contains empty {@code Optional} values in the given
+     * {@code ArgumentMultimap}.
+     */
+    private static boolean isAtLeastOnePrefixPresent(ArgumentMultimap argumentMultimap, Prefix... prefixes) {
```
Comment (alfredkohhh) [inline src/main/java/manageezpz/logic/parser/FindEmployeeCommandParser.java:122]: Thanks for adding this safeguard

File: src/main/java/manageezpz/logic/parser/FindTaskCommandParser.java [hunk 3]
```diff
@@ -58,6 +60,12 @@ public FindTaskCommand parse(String args) throws ParseException {
         }
     }
 
+    private void checkIfHaveAtLeastOneOption(ArgumentMultimap argMultiMap) {
```
Comment (alfredkohhh) [inline src/main/java/manageezpz/logic/parser/FindTaskCommandParser.java:63]: Thanks for adding this safeguard !

File: src/main/java/manageezpz/logic/parser/FindEmployeeCommandParser.java [hunk 4]
```diff
@@ -45,6 +47,12 @@ public FindEmployeeCommand parse(String userInput) throws ParseException {
         }
     }
 
+    private void checkIfHaveAtLeastOneOption(ArgumentMultimap argMultiMap) {
```
Comment (ChanWeiJie) [inline src/main/java/manageezpz/logic/parser/FindEmployeeCommandParser.java:50]: Thank you for doing this check!

File: src/main/java/manageezpz/model/person/PersonMultiplePredicate.java [hunk 5]
```diff
@@ -32,15 +35,15 @@ public PersonMultiplePredicate(List<String> name, String phone, String email) {
...
+        boolean hasName = names != null ? checkIfNameExists(person) : true;
         boolean hasPhone = phone != null ? checkIfPhoneExists(person) : true;
         boolean hasEmail = email != null ? checkIfEmailExists(person) : true;
 
         return hasName && hasPhone && hasEmail;
     }
 
     private boolean checkIfNameExists(Person person) {
-        return name.stream().anyMatch(name -> StringUtil.containsWordIgnoreCase(person.getName().fullName, name));
+        return names.stream().anyMatch(name -> StringUtil.containsWordIgnoreCase(person.getName().fullName, name));
```
Comment (aaron-ljx) [inline src/main/java/manageezpz/model/person/PersonMultiplePredicate.java:46]: good use of stream here

**Review by ChanWeiJie — APPROVED** [review]
LGTM!

**Pull request merged**